### DISCUSSION TITLE
Array of foriegn keys

### DISCUSSION THREAD

**Discussion by cheesycod** [piccolo-orm/piccolo#1244] [General]
It would nice if we could have a array with base_column of foreign keys

**Comment by dantownsend** [comment]
It's an interesting idea. I've never tried creating an array of foreign keys, so not sure what's possible.

**Comment by sinisaos** [comment]
@dantownsend This can already be used if we move `ValueError` check

https://github.com/piccolo-orm/piccolo/blob/16276c68395ebb20d291fdea3bb6cb94e9d2f5ec/piccolo/columns/column_types.py#L2613-L2614

After that the migrations work and we can add records by adding a primary key (without any checking!) to the `base_column` (Piccolo Admin works for adding a primary key (`Serial`, `UUID`), but we lost the [FK select widget](https://piccolo-orm.com/blog/building-a-great-select-widget/)). 
**But should we implement this and what is it useful for?** The Postgres [documentation](https://www.postgresql.org/docs/current/arrays.html) does not mention this option. Also, people advise using many-to-many relationships for this (which makes sense to me) for reasons nicely explained [here](https://stackoverflow.com/a/76502851). I think we should close this as not planned.
